.circleci/config.yml

```diff
@@ -108,298 +95,82 @@ jobs:
       - run:
           name: Verify format
           command: yarn format:check
-      - run:
-          name: Verify lint
-          command: yarn lint:check
       - run:
           name: 'lint package.json files'
           command: 'yarn run packageJsonLint'
-  test-request-client:
+  test-unit:
     docker:
       - *node_image
-      - *ganache_image
+    resource_class: xlarge
     working_directory: *working_directory
     steps:
       - attach_workspace:
           at: *working_directory
       - run:
-          name: 'Build request-client.js'
-          command: 'yarn workspace @requestnetwork/request-client.js run build:commonjs'
-      - run:
-          name: 'Deploy test contracts from smart-contracts'
-          command: 'yarn workspace @requestnetwork/smart-contracts run deploy'
-      - run:
-          name: 'Test request-client.js'
-          command: 'yarn workspace @requestnetwork/request-client.js run test --ci --maxWorkers=1'
```

Comment:
Do all the tests pass, even when parallelized?

In theory, they're supposed to, but I wonder if we serialized them because there were tests that would fail if run in the wrong order.

Reply:
Yes, they do pass you can see in the circle ci run for this PR.